### DISCUSSION TITLE
milvus   restore  fail: "code":3,"msg":"fail to list databases, err: feature not supported"

### DISCUSSION THREAD

**Discussion by 1292253144** [milvus-io/milvus#27409] [Q&A and General discussion]
[ERROR] [core/backup_impl_restore_backup.go:214] ["fail to list databases, err: feature not supported"] [stack="github.com/zilliztech/milvus-backup/core.(*BackupContext).RestoreBackup\n\t/home/runner/work/milvus-backup/milvus-backup/core/backup_impl_restore_backup.go:214\ngithub.com/zilliztech/milvus-backup/core.(*Handlers).handleRestoreBackup\n\t/home/runner/work/milvus-backup/milvus-backup/core/backup_server.go:233\ngithub.com/zilliztech/milvus-backup/core.(*Handlers).RegisterRoutesTo.func6\n\t/home/runner/work/milvus-backup/milvus-backup/core/backup_server.go:120\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/context.go:173\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/recovery.go:101\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/context.go:173\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/logger.go:240\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/context.go:173\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/gin.go:616\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/gin.go:572\nnet/http.serverHandler.ServeHTTP\n\t/opt/hostedtoolcache/go/1.18.10/x64/src/net/http/server.go:2916\nnet/http.(*conn).serve\n\t/opt/hostedtoolcache/go/1.18.10/x64/src/net/http/server.go:1966"]

**Comment by yhmo** [comment]
The latest milvus-backup tool uses ListDatabase() api get database information, but ListDatabase()/CreateDatabase()/DropDatabase() are new feature from Milvus v2.2.7. So, double-check your milvus version.
If your milvus version is lower than v2.2.7, downgrade your mivus-backup tool to v.0.3.x
